### PR TITLE
hotfix on invalid naming of argument

### PR DESCRIPTION
**Pull request by ows090**
Hofix on invalid argument
there is some invalid argument in _showErrorDialog function in auth_screen.dart

### CODE REVIEW

File: lib/screens/auth_screen.dart [hunk 1]
```diff
@@ -167,7 +167,7 @@ class _AuthCardState extends State<AuthCard> {
       _showErrorDialog(errorMessage);
     } catch (error) {
       var errorMessage = 'Could not authenticate you';
-      _showErrorDialog(errorMessage); 
+      _showErrorDialog(errorMessage); // errorMessage go to _showErrorDialog argument
```
Comment (wonseok-second) [inline lib/screens/auth_screen.dart:170]: ok, that's what i want

**Comment by ows090** [comment]
ㅁ

File: lib/screens/auth_screen.dart (hunk 1 above)
Comment (ows090) [reply lib/screens/auth_screen.dart:170]: is there anything to add?

Comment (wonseok-second) [reply lib/screens/auth_screen.dart:170]: no. it's ok

**Pull request merged**